apps/question-bank/src/test/resources/features/page/content/subsection/GroupSubsection.feature

```diff
@@ -0,0 +1,23 @@
+@group_subsection
+Feature: Group Subsection
+
+    Background:
+        Given I have a page
+        And the page has a tab
+        And the page has a section in the 1st tab
+        And the page has a sub-section named "whatever" in the 1st section
+
```

Comment:
Issue: Test does not add any questions to the subsection and therefore does not verify that the question is updated to reflect that it has been grouped / ungrouped.

The method for grouping also filters out non questions which i assume is the same behavior as Classic NBS,
```java
.filter(ui -> batchIds.contains(ui.getId()) && ui.getNbsUiComponentUid() == QUESTION)
```
It would be good to add a static element or other entry to the subsection and verify that it is not modified as part of the grouping / ungrouping.